routingtable/adjRIBIn/adj_rib_in.go

```diff
@@ -64,6 +68,20 @@ func (a *AdjRIBIn) AddPath(pfx net.Prefix, p *route.Path) error {
 	a.mu.Lock()
 	defer a.mu.Unlock()
 
+	// RFC4456 Sect. 8: Ignore route with our RouterID as OriginatorID
+	if p.BGPPath.OriginatorID == 1 {
```

Comment:
i think this should be a equal comparison against our router id insted of a static 1